lib/index.js

```diff
@@ -1,5 +1,9 @@
 /* eslint no-process-env: 0, no-console: 0, max-statements: 0 */
 'use strict';
+process.on('unhandledRejection', function(e) {
+  // no swallowed Promise errors
+  throw e;
```

Comment:
So what happens when this throws? Does that cause an `unhandledException` error to be raised? And if so, what would be the consequences of that?

What problem are we trying to solve here? Will this crash the gulp tasks? It seems like the responsibility of an application to decide how to deal with global unhandled errors, and maybe we wouldn't want whatever this causes in all cases.